include/ck_tile/ops/gemm/block/block_universal_gemm_ar_br_cr.hpp

```diff
@@ -0,0 +1,458 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2025, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/core.hpp"
+#include "ck_tile/ops/gemm/block/block_gemm_areg_bsmem_creg_v1_default_policy.hpp"
+#include "ck_tile/ops/gemm/pipeline/gemm_pipeline_ag_bg_cr_scheduler.hpp"
+#include "ck_tile/ops/elementwise.hpp"
+
+namespace ck_tile {
+
+// A is block distributed tensor
+// B is block distributed tensor
+// C is block distributed tensor
+template <typename Problem_, typename Policy_ = BlockGemmARegBSmemCRegV1DefaultPolicy>
+struct BlockUniversalGemmArBrCr
+{
+    private:
+    // TODO: This should be in Policy - UniversalGemmPolicyBase ?
```

Comment:
With addition of 3 new block gemms and copy of this code 3 times this comment is even more actual.... I think we finally need to do this.